requirements/edx/development.txt

```diff
@@ -256,7 +256,7 @@ pyjwkest==1.3.2
 pyjwt==1.5.2
 pylint-celery==0.3
 pylint-django==0.7.2
-pylint-plugin-utils==0.2.6
+pylint-plugin-utils==0.3
```

Comment:
We'll need to be careful about letting this one continue to auto-upgrade.  From the changelog:

> Ditching Py2 due to pylint having done the same

They didn't do anything to break Python 2.7 in this release, but they've stopped testing it.

Reply:
@jmbowman given that edx-lint supports python 3.5, it seems like this change we want to make is add this pin to `testing.in` with a comment explaining why?

Reply:
Sounds good to me.